docs/hub/security-sso-entra-id-scim.md

```diff
@@ -0,0 +1,111 @@
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/contact/sales?from=enterprise" target="_blank">Enterprise Plus</a> plan.
+</Tip>
+
+### Step 1: Get SCIM configuration from Hugging Face
+
+1.  Navigate to your organization's settings page on Hugging Face.
+2.  Go to the **SSO** tab, then click on the **SCIM** sub-tab.
+3.  Copy the **SCIM Tenant URL**. You will need this for the Entra ID configuration.
+4.  Click **Generate an access token**. A new SCIM token will be generated. Copy this token immediately and store it securely, as you will not be able to see it again.
+
+<div class="flex justify-center">
+    <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/sso/scim-settings.png"/>
+    <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/sso/scim-settings-dark.png"/>
+</div>
+
+### Step 2: Configure Provisioning in Microsoft Entra ID
+
+1.  In the Microsoft Entra admin center, navigate to your Hugging Face Enterprise Application.
+2.  In the left-hand menu, select **Provisioning**.
+3.  Click **Get started**.
+4.  Change the **Provisioning Mode** from "Manual" to **Automatic**.
+
+### Step 3: Enter Admin Credentials
+
+1.  In the **Admin Credentials** section, paste the **SCIM Tenant URL** from Hugging Face into the **Tenant URL** field.
+2.  Paste the **SCIM token** from Hugging Face into the **Secret Token** field.
+3.  Click **Test Connection**. You should see a success notification.
+4.  Click **Save**.
+
+<div class="flex justify-center">
+    <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/sso/scim-entra-creds.png" alt="Entra ID SCIM Admin Credentials"/>
+    <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/sso/scim-entra-creds-dark.png" alt="Entra ID SCIM Admin Credentials"/>
+</div>
+
+### Step 4: Configure Attribute Mappings
+
+1.  Under the **Mappings** section, click on **Provision Microsoft Entra ID Users**.
+2.  The default attribute mappings often require adjustments for robust provisioning. We recommend using the following configuration. You can delete attributes that are not listed here:
+
+    | `customappsso` Attribute | Microsoft Entra ID Attribute | Matching precedence | Actions |
+    |---|---|---|---|
+    | `userName` | `Replace([mailNickname], ".", "", "", "", "", "")` | | |
+    | `active` | `Switch([IsSoftDeleted], , "False", "True", "True", "False")` | | `Delete` |
+    | `emails[type eq "work"].value` | `userPrincipalName` | | |
+    | `name.givenName` | `givenName` | | `Edit`, `Delete` |
+    | `name.familyName` | `surname` | | `Edit`, `Delete` |
+    | `name.formatted` | `Join(" ", [givenName], [surname])` | | |
+    | `externalId` | `objectId` | `1` | |
```

Comment:
```suggestion
    | `customappsso` Attribute | Microsoft Entra ID Attribute | Matching precedence | 
    |---|---|---|
    | `userName` | `Replace([mailNickname], ".", "", "", "", "", "")` | |
    | `active` | `Switch([IsSoftDeleted], , "False", "True", "True", "False")` | | 
    | `emails[type eq "work"].value` | `userPrincipalName` | |
    | `name.givenName` | `givenName` | |
    | `name.familyName` | `surname` | |
    | `name.formatted` | `Join(" ", [givenName], [surname])` | |
    | `externalId` | `objectId` | `1` |
```

Actions column is useless, bad paste from transcribing the screen to markdown :p